model/raw/src/main/scala/aqua/raw/value/ValueRaw.scala

```diff
@@ -185,6 +185,19 @@ case class CollectionRaw(values: NonEmptyList[ValueRaw], boxType: BoxType) exten
     copy(values = values.map(_.renameVars(map)))
 }
 
+case class DataRaw(name: String, fieldsList: NonEmptyList[(String, ValueRaw)], baseType: Type) extends ValueRaw {
```

Comment:
This name does not speak. Maybe `MakeStructRaw`? So that it's closer to `CallArrowRaw`.

What is `name`? Why `Data` has the name?